WTWSMS/common/scripted_triggers/WtWSMS_scripted_triggers.txt

```diff
@@ -28,6 +28,7 @@ has_romance_culture_trigger = {
 		culture = helleno_thracian
 		culture = massalian
 		culture = romano_dacian
+		culture = spartan
```

Comment:
I agree that Spartans shouldn't be able to create Legions or restore them, but I wanted Spartans to have a chance to form Rome.